app/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+apply from: '../autodimension.gradle'
+
+/**
+ * start config and check ci*/
+apply from: '../checkci.gradle'
+
+android {
+  compileSdkVersion 26
+  buildToolsVersion "25.0.3"
+  defaultConfig {
+    applicationId "ci.sample.toandoan.circlecisample"
+    minSdkVersion 14
+    targetSdkVersion 26
+    versionCode 1
+    versionName "1.0"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    }
+  }
+}
+
+dependencies {
+  compile fileTree(dir: 'libs', include: ['*.jar'])
+  androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    exclude group: 'com.android.support', module: 'support-annotations'
+  })
+  compile 'com.android.support:appcompat-v7:26.+'
```

Comment:
WARNING: [GradleDynamicVersion] Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (com.android.support:appcompat-v7:26.+)
 Using `+` in dependencies lets you automatically pick up the latest available version rather than a specific, named version. However, this is not recommended; your builds are not repeatable; you may have tested with a slightly different version than what the build server used. (Using a dynamic version as the major version number is more problematic than using it in the minor version position.)